AmahiAnywhere/AmahiAnywhere/MenuTableViewCell.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MenuTableViewCell.swift
+//  AmahiAnywhere
+//
+//  Created by Syed on 24/03/18.
+//  Copyright © 2018 Amahi. All rights reserved.
+//
+
+import UIKit
+
+class MenuTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var lblMenuName: UILabel!
```

Comment:
No need to add Menu in the variable name this is already a menu cell. titleLabel is more appropriate. Generally, avoid using abbreviations, you should use a full description of the UI widget as a subscript of your variable names. Check out naming conventions in the swift styles shared earlier https://github.com/raywenderlich/swift-style-guide